src/three-pathfinding.d.ts

```diff
@@ -0,0 +1,131 @@
+import type { BufferGeometry, Object3D, Vector3 } from "three";
+
+declare module "three-pathfinding" {
```

Comment:
Is the 'declare module'  block required? In other some packages I maintain — written in TypeScript, and compiled with Microbundle — the generated type definitions don't have this block, so I just want to make sure I understand the need, if any. :)

Reply:
This was not necessary for the 'declare module' block. I am very sorry about that.